app/index.html

```diff
@@ -5,9 +5,10 @@
   <title>Darts Tournament App</title>
   <link rel="stylesheet" href="../bower_components/bootstrap/dist/css/bootstrap.min.css" />
   <link rel="stylesheet" href="app.css">
+  <link rel="stylesheet" href="player/player.css">
```

Comment:
Of course this approach introduces a lot of side effects. Did you notice that `app.css` is automatically generated by `gulp`. As mentioned in a previous comment, you should add your css rules in some `*.scss` files that are added in the specific folder. The structure of the folders follows the `SMACSS` approach.

Please remove this occurrences and update your code in order to be automatically added your css into `app.css`